src/chapters/aspdotnet-model-validation/viewmodels.rst

```diff
@@ -0,0 +1,91 @@
+.. index:: ! ViewModel
+
+ViewModels and Passing Data To and From Views
+=============================================
+
+Now that we have an understanding for what a model is, we can focus on how to effectively pass information between the three elements of MVC applications.
+With our current MVC application, we can add new events and remove events.
+However, our application is also susceptible to run-time errors.
+Our view can accept any type of input and if we mistype something in our view, we can run into issues later down the line.
+
+A **ViewModel** is a model class designed specifically to be used in a view.
+By utilizing ViewModels in our application, we can make our views strongly-typed and add validation to forms to prevent bad user input.
+These benefits of ViewModels will help reduce potential errors in our application.
+
+Refactoring the View and Controller to Use a Model in a View - Video
+--------------------------------------------------------------------
+
+.. TODO: Add video covering an intro about how to use a model in a view
+
+.. admonition:: Note
+
+   If you want to verify what code this video starts with, check out the `model-binding <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/model-binding>`__ branch.
+   If you want to verify what code this video ends with, check out the `models-in-views <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/models-in-views>`__ branch.
+
+Refactoring the View and Controller to Use a Model in a View - Text
+-------------------------------------------------------------------
+
+To start with understanding why we may want to use a ViewModel, let's refactor our code to use a model directly in our view.
+
+First, we want to convert the collection of Events we have been holding into a list.
+
+.. sourcecode:: csharp
+   :lineno-start: 19
+
+   List<Event> events = new List<Event>(EventData.GetAll());
+
+Now that we are storing our items in a ``List``, we need to import the model into our ``Events/Index.cshtml`` view so we can use the new ``events`` collection.
+We can add a small statement up on line 1 to do so: 
+
+.. sourcecode:: csharp
+
+   @model List<CodingEventsDemo.Models.Event>
+
+Wherever we used our ``ViewBag`` property, we can now use ``Model`` syntax.
+Once the view has been updated, run the application!
+
+This was merely a refactor so the functionality of the app hasn't changed, but we have eliminated some of the possibility of bugs in our code being discovered at runtime!
```

Comment:
It's not clear to me at this point why it's less likely to have bugs now. Is this because there's compile-time checking of any references to properties of the model?

Reply:
It does do compile-time checking! Added more detail to the text.